src/include/homestore/index/index_table.hpp

```diff
@@ -147,6 +148,11 @@ class IndexTable : public IndexTableBase, public Btree< K, V > {
         for (const auto& right_child_node : new_nodes) {
             auto right_child = IndexBtreeNode::convert(right_child_node.get());
             write_node_impl(right_child_node, context);
+#ifdef _PRERELEASE
+            if (iomgr_flip::instance()->test_flip("index_right_sibling")) {
```

Comment:
Would suggest a more contextful name for the flip, say `crash_on_right_sibling_flush`, 
`crash_on_left_sibling_flush`, 
`crash_on_non_root_parent_flush`, 
`crash_on_root_parent_flush`